test/cloud.jl

```diff
@@ -0,0 +1,13 @@
+using Base.Test
+using Gurobi
+using MathProgBase.SolverInterface
+
+env = Gurobi.Env("csdemo.gurobi.com")
+Gurobi.free_env(env)
```

Comment:
@czlee, you can run the [mathprog](https://github.com/JuliaOpt/Gurobi.jl/blob/master/test/mathprog.jl) tests and just plugin the cloud solver.